src/partials/footer.html

```diff
@@ -1 +1,30 @@
-<footer>TEST</footer>
+<footer>
+  <div>
+    <div>HOMEMADE ICE CREAM MADE<span> WITH PASSION</span></div>
+    <div>
+      <ul>
+        <li>
+          <a target="_blank"
+            ><svg><use>git</use></svg></a
+          >
+        </li>
+        <li>
+          <a target="_blank"
+            ><svg><use>tel</use></svg></a
+          >
+        </li>
+        <li>
+          <a target="_blank"
+            ><svg><use>fb</use></svg></a
+          >
+        </li>
+        <li>
+          <a target="_blank" href="tel.+610383766284"
+            >Feel free to contact <span>+61 (0) 383 766 284</span></a
+          >
+        </li>
+      </ul>
+    </div>
+    <div>&copy; 2019 - IceCream. All Rights Reserved.</div>
```

Comment:
Najlepiej użyć tutaj zwykły tag span

Reply:
dlaczego nie może zostać div? co to za różnica? Oba tagi są elementami blokowymi.

Reply:
semantyka